feugy/tabulous
# feat(web): implements public catalog with anonymous access

### What's in there?

Now that copyrighted games are not freely accessible, it's time to implement a public home page that only displays publicly available items.
Starting a new game requires authentication.

- feat(web): implements public catalog with anonymous access
- chore(web): migrates to latest svelte-kit which uses vite commands and config instead of its own wrappers.

### How to test?

1. start the application
2. browse to https://localhost:3000
   > you should be presented a page with generic title, a link to login page, no started games and a catalog of 2 items
3. click on any of the catalog items
   > you should be brought to the login page
4. log in with credentials
   > you should be redirected to the game's page
5. go back to home
   > the home page now contains a new title, a link to logout, the started game and the catalog

Various page refresh (authenticated or not) should stay consistent.
